codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/BuilderGenerator.kt

```diff
@@ -136,8 +136,7 @@ class BuilderGenerator(
         member: MemberShape,
         memberName: String
     ) {
-        // Render a `set_foo` method. This is useful as a target for code generation, because the argument type
-        // is the same as the resulting member type, and is always optional.
+        // Render a `set_foo` method.
```

Comment:
Looks like I removed this comment by accident in my commit

Reply:
Will add it back.